README.md

```diff
@@ -28,9 +28,59 @@ import { NgxSherlockModule } from '@politie/ngx-sherlock';
 export class AppModule { }
 ```
 
+### `autoDetectChanges`
+
+**Signature**:
+```typescript
+autoDetectChanges(detectorRef: ChangeDetectorRef): void;
+```
+
+The function `autoDetectChanges` will automatically run a change detection cycle when `Derivables` or `DerivableProxies` within a component's template change internal state. The function should be called in an `OnInit` lifecycle hook of a component or directive.
+
+The `autoDetectChanges` function guarantees model and view fidelity, meaning one can easily use Angular's forms and template functionality.
+
+#### Example
+
+`trusty-sidekick.component.html`:
+```html
+<h2>Well there you are, {{sidekick$.firstname.$value}} {{sidekick$.surname.$value}}!</h2>
+<input type="text" [(ngModel)]="sidekick$.firstname.$value" placeholder="First name">
```

Comment:
This example is very clear. It would also work without `autoDetectChanges` though, because `ngModel` kicks off a change detection on every keystroke, so every change is processed anyway. Maybe split the code into two components, but leave out most boilerplate to keep it concise.

Reply:
I really like "trusty-sidekick" 😄